src/components/tab-bar/tab-bar.tsx

```diff
@@ -0,0 +1,139 @@
+import { Component, h, Prop, Element, State } from '@stencil/core';
+import {
+    MDCTabBar,
+    MDCTabBarActivatedEvent
+} from '@limetech/mdc-tab-bar';
+import { strings } from '@limetech/mdc-tab-bar/constants';
+import { Tab } from './tab.types';
+const { TAB_ACTIVATED_EVENT } = strings;
+
+@Component({
+    tag: 'limel-tab-bar',
+    styleUrl: 'tab-bar.scss',
+    shadow: true
+})
+export class TabBar {
```

Comment:
Since this will probably be worked on for a while during the cycle, we can add a feature flag that will prevent it from being used until we are ready to release it :slightly_smiling_face: